.travis.yml

```diff
@@ -3,12 +3,12 @@ dist: trusty
 sudo: true
 
 env:
-  - NOVALGRIND=1
-  - NOVALGRIND=0
+  - NO_VALGRIND=1
```

Comment:
I sort of liked having the ability to disable valgrind selectively, it allowed us to run part of the tests quicker in one CI build and slower on the other. Merging the two makes one CI run really short and the other even longer. But I agree that if that is the aim we should pick more descriptive variable names.